### ISSUE TITLE
Link Purpose Should be more descriptive to users for "See PR" Link in Announcing TypeChat 0.1.0 Page: A11y_TypeChat_Docs Link_Introduction screen_Link Purpose

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/TypeChat#280]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #TypeChat; #BM_TypeChat_Web_Nov2025; #Win11; #Link purpose; #WCAG2.4.4; #FTP; #DesktopWeb; #A11yMAS; #ChromiumEdge;.

### Environment and OS details:
URL: https://microsoft.github.io/TypeChat/
Windows 11
OS build: 26200.6899
Chromium Edge Version 142.0.3595.53 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL: https://microsoft.github.io/TypeChat/
2. Typechat Home screen gets opened.
3. Tab Navigate to "Blogs Link".
4. Introduction Typechat page gets opened.
5. Tab navigate to Announcing TypeChat 0.1.0 link and press enter on it.
6. Turn on Screen Reader and Observe that Link Purpose Should be more descriptive to users for "See PR" Link in Announcing TypeChat 0.1.0 Page.

### Actual Result:
Link Purpose Should be more descriptive to users for "See PR" Link in Announcing TypeChat 0.1.0 Page.

### Expected Result:
See PR link should be more descriptive to users.

### User Impact:
Screen reader users may encounter challenges if the purpose of the "See PR" link on the Announcing TypeChat 0.1.0 page is not described in more detail. Providing a more descriptive link would help users better understand its function and improve accessibility for everyone. Clearer link text is especially important for those relying on assistive technologies.

### Attachment:

https://github.com/user-attachments/assets/a222a75a-2c00-4dd3-8996-1f29f31a5399
